### PR TITLE
[hotfix] modify sitemap, add robots.txt and fix /toc + test perf. improvements

### PR DESCRIPTION
**Pull request by emanuil-tolev**
#1130

### CODE REVIEW

**Comment by emanuil-tolev** [comment]
@Steven-Eardley would you have a minute to review this this week? Richard's probably going to be working on other things. Hopefully it's not too bad.

**Comment by Steven-Eardley** [comment]
Will do, tomorrow morning I expect

File: portality/view/doaj.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def autocomplete(doc_type, field_name):
 
 @blueprint.route("/toc")
 def list_journals():
-    js = models.Journal.all_in_doaj(page_size=1000, minified=True)
```
Comment (Steven-Eardley) [inline portality/view/doaj.py]: Why the demise of minified? Apart from the old ```all_in_doaj``` code being confusing?

Comment (emanuil-tolev) [inline portality/view/doaj.py]: Two things for me:

1. it was not being used anywhere else
2. it did not return model objects - just JSON ones with a few values (the point of minified). We actually need to use a method in the template serving this endpoint, `.toc_id()`, so in fact we do end up needing the whole object. I just didn't see any point in it after that - but I may have missed something!

Comment (Steven-Eardley) [inline portality/view/doaj.py]: I think I'm with you there, thanks

**Pull request closed without merging**